### PR TITLE
feat: support celo

### PR DESCRIPTION
**Pull request by Jesse-Sawa**
This change also includes removing lists from the repository.
 
Screenshots of Celo widget support using the interface
Screen Shot 2022-10-20 at 11 55 25 AM

### CODE REVIEW

File: src/constants/addresses.ts [hunk 1]
```diff
@@ -5,6 +5,10 @@ type AddressMap = { [chainId: number]: string }
 
 export const UNI_ADDRESS: AddressMap = constructSameAddressMap('0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984')
 
+// celo v3 addresses
+const CELO_ROUTER_ADDRESS = '0x5615CDAb10dc425a742d643d949a7F474C01abc4'
```
Comment (zzmp) [inline src/constants/addresses.ts:9]: Just inline these to keep this file simpler, even though they'll be repeated for CELO and CELO_ALFAJORES.

File: src/utils/getExplorerLink.ts [hunk 2]
```diff
@@ -10,6 +10,8 @@ const ETHERSCAN_PREFIXES: { [chainId: number]: string } = {
   [SupportedChainId.OPTIMISTIC_KOVAN]: 'https://kovan-optimistic.etherscan.io',
   [SupportedChainId.POLYGON_MUMBAI]: 'https://mumbai.polygonscan.com',
   [SupportedChainId.POLYGON]: 'https://polygonscan.com',
+  [SupportedChainId.CELO]: 'https://celoscan.io',
+  [SupportedChainId.CELO_ALFAJORES]: 'https://alfajores.celoscan.io/',
```
Comment (zzmp) [inline src/utils/getExplorerLink.ts:14]: rm the trailing `/`

File: src/utils/getExplorerLink.test.ts [hunk 3]
```diff
@@ -19,6 +19,9 @@ describe('#getExplorerLink', () => {
   it('polygon', () => {
     expect(getExplorerLink(137, 'abc', ExplorerDataType.ADDRESS)).toEqual('https://polygonscan.com/address/abc')
   })
+  it('celo', () => {
```
Comment (zzmp) [inline src/utils/getExplorerLink.test.ts:22]: add alfajores

File: src/constants/routing.ts [hunk 4]
```diff
@@ -163,6 +169,19 @@ export const COMMON_BASES: ChainCurrencyList = {
     WRAPPED_NATIVE_CURRENCY[SupportedChainId.POLYGON_MUMBAI] as Token,
     WETH_POLYGON_MUMBAI,
   ],
+  [SupportedChainId.CELO]: [
+    nativeOnChain(SupportedChainId.POLYGON),
```
Comment (zzmp) [inline src/constants/routing.ts:173]: SupportedChainId.CELO

File: src/constants/routing.ts [hunk 5]
```diff
@@ -163,6 +169,19 @@ export const COMMON_BASES: ChainCurrencyList = {
...
+    WETH_POLYGON,
```
Comment (zzmp) [inline src/constants/routing.ts:174]: These should be

    CEUR_CELO,
    CUSD_CELO,
    PORTAL_ETH_CELO,
    PORTAL_USDC_CELO,
    CMC02_CELO,

File: src/constants/routing.ts [hunk 6]
```diff
@@ -163,6 +169,19 @@ export const COMMON_BASES: ChainCurrencyList = {
...
+    USDC_POLYGON,
+    DAI_POLYGON,
+    USDT_POLYGON,
+    WBTC_POLYGON,
+  ],
+  [SupportedChainId.CELO_ALFAJORES]: [
+    nativeOnChain(SupportedChainId.POLYGON_MUMBAI),
```
Comment (zzmp) [inline src/constants/routing.ts:181]: SupportedChainId.CELO_ALFAJORES

File: src/constants/routing.ts [hunk 7]
```diff
@@ -163,6 +169,19 @@ export const COMMON_BASES: ChainCurrencyList = {
...
+    WRAPPED_NATIVE_CURRENCY[SupportedChainId.POLYGON_MUMBAI] as Token,
```
Comment (zzmp) [inline src/constants/routing.ts:182]: These should be

    CUSD_CELO_ALFAJORES,
    CEUR_CELO_ALFAJORES,

File: src/constants/tokens.ts [hunk 8]
```diff
@@ -316,6 +388,35 @@ export const WRAPPED_NATIVE_CURRENCY: { [chainId: number]: Token | undefined } =
     'WMATIC',
     'Wrapped MATIC'
   ),
+  [SupportedChainId.CELO]: new Token(
```
Comment (zzmp) [inline src/constants/tokens.ts:391]: Why not re-use the tokens you defined earlier in this file (L282, L317)?

File: src/hooks/useCurrencyLogoURIs.ts [hunk 9]
```diff
@@ -36,6 +42,13 @@ function getTokenLogoURI(address: string, chainId: SupportedChainId = SupportedC
   if (networksWithUrls.includes(chainId)) {
     return `https://raw.githubusercontent.com/Uniswap/assets/master/blockchains/${networkName}/assets/${address}/logo.png`
   }
+
+  // Celo logo is hosted elsewhere.
+  if (isCelo(chainId)) {
+    if (address === nativeOnChain(chainId).wrapped.address) {
+      return 'https://raw.githubusercontent.com/ubeswap/default-token-list/master/assets/asset_CELO.png'
```
Comment (zzmp) [inline src/hooks/useCurrencyLogoURIs.ts:49]: Why is this hosted on ubeswap? Does celo have a canonical file that we can store and reference?

**Review by zzmp — CHANGES_REQUESTED** [review]

File: src/constants/routing.ts (hunk 5 above)
Comment (Jesse-Sawa) [reply src/constants/routing.ts:174]: 🫠

File: src/hooks/useCurrencyLogoURIs.ts (hunk 9 above)
Comment (Jesse-Sawa) [reply src/hooks/useCurrencyLogoURIs.ts:49]: Yea, this is copy pasta from the interface back when I didn't have a 'celo-org' hosted token list 
I'll replace it with 'https://celo-org.github.io/celo-token-list/assets/celo_logo.svg''

File: src/hooks/useCurrencyLogoURIs.ts [hunk 10]
```diff
@@ -46,7 +46,7 @@ function getTokenLogoURI(address: string, chainId: SupportedChainId = SupportedC
   // Celo logo is hosted elsewhere.
   if (isCelo(chainId)) {
     if (address === nativeOnChain(chainId).wrapped.address) {
-      return 'https://raw.githubusercontent.com/ubeswap/default-token-list/master/assets/asset_CELO.png'
+      return 'https://celo-org.github.io/celo-token-list/assets/celo_alternative_logo.png'
```
Comment (zzmp) [inline src/hooks/useCurrencyLogoURIs.ts:49]: Why is this the _alternative_ logo?

**Review by zzmp — CHANGES_REQUESTED** [review]

File: src/hooks/useCurrencyLogoURIs.ts (hunk 10 above)
Comment (Jesse-Sawa) [reply src/hooks/useCurrencyLogoURIs.ts:49]: The alternative logo (yellow and white) is used for erc-20 transfers and is consistent with other applications on Celo. The [green and yellow](https://celo-org.github.io/celo-token-list/assets/celo_logo.svg) icon normally represents the network itself. 

For UI consistency; Celo on the celo token list (and others alike) use the alternative logo to represent $CELO. If $CELO is selected from the token list, the same logo should be displayed on the swap widget.

**Review by zzmp — APPROVED** [review]
Approved pending a comment inline to explain why you are using the alternative logo (ie brand guideline reference).

**Pull request merged**